PiwikTracker/Tracker.swift

```diff
@@ -63,4 +65,60 @@ final public class Tracker: NSObject {
         }
         self.dispatchTimer = Timer.scheduledTimer(timeInterval: dispatchInterval, target: self, selector: #selector(dispatch), userInfo: nil, repeats: false)
     }
+    
+    var visitor = Visitor.current()
+    var session = Session.current()
+}
+
+// shared instance
+extension Tracker {
+    public static var sharedInstance: Tracker? {
+        get { return _sharedInstance }
+    }
+    
+    public class func configureSharedInstance(withSiteID siteID: String, baseURL: URL) {
+        let queue = MemoryQueue()
+        let dispatcher = URLSessionDispatcher(baseURL: baseURL)
+        self._sharedInstance = Tracker.init(siteId: siteID, queue: queue, dispatcher: dispatcher)
```

Comment:
Somehow this feels kind of strange setting the `_sharedInstance` in a class function... But I don't have a better solution for now.